smac/facade/smac_facade.py

```diff
@@ -32,7 +33,9 @@ class SMAC(object):
 
     def __init__(self,
                  scenario: Scenario,
-                 tae_runner: ExecuteTARun=None,
+                 # TODO: once we drop python3.4 add type hint
```

Comment:
I don't like such TODOs which will maybe never resolved. Either cut it or do it right away.